docs/reference/field-types/dateAndTime.md

```diff
@@ -0,0 +1,40 @@
+# `dateAndTime`
+
+`dateAndTime` fields are text field with UI support, and limitation, for saving date and time values. Date and times are stored in ISO 8601 format. `YYYY-MM-DDTHH:MM:SSZ` format.
```

Comment:
"Dates and times are stored in ISO 8601 format, for example: 2022-01-01T03:00:00Z

Note that while dates and times are edited in the individual editor's time zone, they are always saved in UTC (Universal Coordinated Time)."